app/controllers/admin/publishers_controller.rb

```diff
@@ -4,7 +4,7 @@ class Admin::PublishersController < AdminController
   include ActiveRecord::Sanitization::ClassMethods
 
   def index
-    ActiveRecord::Base.connected_to(role: :reading) do
+    ActiveRecord::Base.connected_to(database: :primary) do
```

Comment:
Since it's by default connected to primary, just remove `ActiveRecord::Base.connected_to(role: :reading) do` completely

Reply:
Good call, I assumed we did auto-switching, but we don't.